src/osemgrep/networking/Semgrep_login.ml

```diff
@@ -17,7 +17,8 @@
 (* Types *)
 (*****************************************************************************)
 
-type login_session = Uuidm.t * Uri.t
+type shared_secret = Uuidm.t
```

Comment:
Many of the PA people are not super familiar with the world of web apps, and auth, and uuidm, so it might
be useful to explain in a git comment the workflow and what is the difference between
the user_anonymous_id, a shared_secret, a session etc.
and what's the goal of this shared secret, what does it improve.

Reply:
Imagine that most PA people are 5 years old ...